django-stubs/core/management/commands/compilemessages.pyi

```diff
@@ -0,0 +1,13 @@
+from django.core.management.base import BaseCommand as BaseCommand, CommandError as CommandError, CommandParser as CommandParser
+from django.core.management.utils import find_command as find_command, popen_wrapper as popen_wrapper
+from typing import Any, List
+
+def has_bom(fn: Any): ...
+def is_writable(path: Any): ...
+
+class Command(BaseCommand):
+    program: str = ...
+    program_options: List[str] = ...
+    verbosity: int = ...
+    has_errors: bool = ...
+    def compile_messages(self, locations: Any) -> None: ...
```

Comment:
Let's fix a bit of `Any` types! I am pretty sure that `locations` cannot be `Any`, can they?